plugins/influxdb/database_credentials.go

```diff
@@ -0,0 +1,40 @@
+package influxdb
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func DatabaseCredentials() schema.CredentialType {
+	return schema.CredentialType{
+		Name:    credname.DatabaseCredentials,
+		DocsURL: sdk.URL("https://docs.influxdata.com/influxdb/v2.7/reference/cli/influx/config/create/"),
```

Comment:
Let's update this link to https://docs.influxdata.com/influxdb/cloud/reference/cli/influx/config/ because the shell plugin seems to be targeted primarily at InfluxDB Cloud (InfluxDB Cloud 2.0 and Enterprise too, I think) users and not InfluxDB OSS.

Ideally, we'd like to support InfluxDB OSS too but I am going to take a closer look at the differences soon.

Reply:
[Latest discussion is here](https://github.com/1Password/shell-plugins/pull/332#pullrequestreview-1524119628).